db/schema.cds

```diff
@@ -22,8 +22,11 @@ entity Incidents : cuid, managed {
   title         : String @title: 'Title';
   urgency       : Association to Urgency;
   status        : Association to Status;
-  conversations : Composition of many Conversations
-                    on conversations.incidents = $self;
+  conversation  : Composition of many {
+    key timestamp : type of managed:createdAt;
+    author        : type of managed:createdBy;
```

Comment:
Having conversations as an inline entity still breaks Fiori Annotation Modeling and potentially LCAP? Need to follow up on that..
With the current story, where the incident processor records the conversation for both customer and processor having the author as createdBy doesn't work, as we can't have any conversations by the customer in this scenario
Initially the timestamp was also intentionally not $now, because we wanted to allow to record a conversation with the customer that happened in the past.